platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/state/RandomSignedStateGenerator.java

```diff
@@ -199,6 +206,9 @@ public SignedState build() {
             v.setConsensusTimestamp(consensusTimestampInstance);
         });
 
+        FAKE_MERKLE_STATE_LIFECYCLES.initRosterState((MerkleStateRoot) stateInstance);
+        // TODO: future work: populate roster history and stop setting AddressBook on platformState above.
```

Comment:
The term `TODO` gets caught by some code scanners.  We (the former platform hashgraph team) have  been using just `Future Work` or other variants to indicate work that will be completed in future PRs and leave `TODO` as something that needs to be completed before a PR can be merged.   Is this needing to be completed before this PR can be merged?